iroha_client_cli/src/main.rs

```diff
@@ -0,0 +1,307 @@
+use clap::{App, Arg};
+
+const CONFIG: &str = "config";
+const CREATE: &str = "create";
+const UPDATE: &str = "update";
+const GET: &str = "get";
+
+fn main() {
+    let matches = App::new("Iroha CLI Client")
+        .version("0.1.0")
+        .author("Nikita Puzankov <puzankov@soramitsu.co.jp>")
```

Comment:
Should we change author to `Soramitsu`? Just thinking what the company policy is on this :)